microsoft/testsuites/ltp/ltp.py

```diff
@@ -125,7 +125,8 @@ def run_test(
         self.run(parameters, sudo=True, force_run=True, timeout=12000)
 
         # to avoid no permission issue when copying back files
-        self.node.tools[Chmod].update_folder("/opt", "a+rwX", sudo=True)
+        self.node.tools[Chmod].update_folder("/opt/ltp", "a+rwX", sudo=True)
+        self.node.tools[Chmod].update_folder("/home/lisatest", "a+rwX", sudo=True)
```

Comment:
I think this should already be present. Can you please confirm?

Reply:
What should already be present?

Reply:
The read/write permission on the lisatest folder. Can you please confirm?

Reply:
Oh I tried it without this at first and got a similar permissions error. I will retest without it to confirm though.

Reply:
If this gives an error then it should be a bug. We use `lisatest` credential to access this folder and it's home folder should be accessible by itself.

Reply:
I retested and it failed with a similar permissions error to before my change.

Reply:
So should I file a new bug report for that or should I try to address it in this PR?

Reply:
What's the permission error here? Can you share from the log?

Reply:
PermissionError: [Errno 13] Permission denied

Reply:
Which commands get this error? Is there other outputs from this command?
BTW, don't assume the user account is always "lisatest". It's configurable. It needs to use `~` to expand to home folder.

Reply:
@squirrelsc The account LISA is using should have read/write access to this folder, right? And it should be something that should be handled by framework rather than individual tests?

Reply:
Can you double check if it causes by the source file not exist, or dest folder not exist?

Reply:
for clarity you want me to check to see if the error is reproduced with a missing source or destination file?

Reply:
Yes, I want to double check where the error message is from. My understanding is the home folder shouldn't have any permission issue to copy a file back.

Reply:
With a missing source file the error changes to say "FileNotFoundError: [Errno 2] No such file" so it is not a missing file issue but definitely a permissions issue.